client/src/app/site/pages/meetings/pages/agenda/modules/topics/modules/topic-poll/components/topic-poll-dialog/topic-poll-dialog.component.html

```diff
@@ -1,79 +1,81 @@
+                        <!-- implicit item references into the component using ng-template slot -->
+                        <ng-template let-item>
+                            <div class="single-option-line" #item>
+                                <span>{{ item.getTitle() }}</span>
+                                <span class="delete-button">
+                                    <button
+                                        mat-icon-button
+                                        matTooltip="{{ 'Remove option' | translate }}"
+                                        (click)="removeOption(item)"
+                                    >
+                                        <mat-icon>clear</mat-icon>
+                                    </button>
+                                </span>
+                            </div>
+                        </ng-template>
+                    </os-sorting-list>
                 </div>
 
-                <!-- Add option -->
                 <div>
+                    <!-- Enter new option -->
+                    <div>
+                        <mat-form-field>
+                            <mat-label>{{ 'New option' | translate }}</mat-label>
+                            <input require matInput [(ngModel)]="optionInput" type="text" />
+                            <mat-error>{{ 'This field is required.' | translate }}</mat-error>
+                        </mat-form-field>
+                    </div>
+
+                    <!-- Add option -->
                     <div>
-                        <button mat-button color="accent" (click)="addNewOption()" [disabled]="optionInput === ''">
-                            <mat-icon>add</mat-icon>
-                            <span>{{ 'Add option' | translate }}</span>
-                        </button>
+                        <div>
+                            <button mat-button color="accent" (click)="addNewOption()" [disabled]="optionInput === ''">
+                                <mat-icon>add</mat-icon>
+                                <span>{{ 'Add option' | translate }}</span>
+                            </button>
+                        </div>
                     </div>
                 </div>
             </div>
-        </div>
-    </ng-container>
-
-    <!-- Too few options warning -->
-    <div class="potential-warning">
-        <ng-container *ngIf="!hasEnoughOptions()">
-            <i class="foreground-warn">
-                {{ optionsWarning | translate }}
-            </i>
         </ng-container>
+
+        <!-- Too few options warning -->
+        <div class="potential-warning">
+            <ng-container *ngIf="!hasEnoughOptions()">
+                <i class="foreground-warn">
+                    {{ optionsWarning | translate }}
+                </i>
+            </ng-container>
+        </div>
     </div>
 </div>
 
 <!-- Actions -->
-<div mat-dialog-actions>
+<div mat-dialog-actions class="padding-left-24 padding-top-bottom-16">
```

Comment:
I am not sure if we should do this. In my opinion we should just stick to the defaults from mdc as much as possible. 

In future versions the appearance could always change and adding imho unnecessary adaptions should be prevented.